domains/hex4.json

```diff
@@ -5,7 +5,9 @@
     },
     "record": {
         "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "MX": ["mx1.forwardemail.net"],
+        "MX": ["mx2.forwardemail.net"],
+        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all",
+        "TXT": "forward-email=nenex54@pm.me"
```

Comment:
TXT records should be in an array btw.

Reply:
Ah, okay. Does that go for MX records as well?

Reply:
Yes.